From bf25d52dbf7cc7ac6cf687c0c1d9e44fd2ee5acc Mon Sep 17 00:00:00 2001
From: Daniel Abrecht <public@danielabrecht.ch>
Date: Fri, 31 Dec 2021 22:48:13 +0100
Subject: [PATCH] Make touchscreen size a bit smaller, to ensure the edges can
 be reached.

I can't quiet reach the right side and bottom of the screen.
So let's pretend it's a few px smaller in both directions.
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index f4073d9a6fca..32f2a87936c9 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -833,8 +833,8 @@ touchscreen@14 {
 		reset-gpios = <&gpio3 RK_PB4 GPIO_ACTIVE_HIGH>;
 		AVDD28-supply = <&vcc3v0_touch>;
 		VDDIO-supply = <&vcc3v0_touch>;
-		touchscreen-size-x = <720>;
-		touchscreen-size-y = <1440>;
+		touchscreen-size-x = <710>;
+		touchscreen-size-y = <1435>;
 		poweroff-in-suspend;
 	};
 
-- 
2.30.2

